src/nvim/testdir/test_execute_func.vim

```diff
@@ -82,3 +84,43 @@ func Test_execute_not_silent()
   endfor
   call assert_equal('xyz ', text2)
 endfunc
+
+func Test_win_execute()
+  let thiswin = win_getid()
+  new
+  let otherwin = win_getid()
+  call setline(1, 'the new window')
+  call win_gotoid(thiswin)
+  let line = win_execute(otherwin, 'echo getline(1)')
+  call assert_match('the new window', line)
+
```

Comment:
https://github.com/vim/vim/commit/868b7b6712ea4f2232eeeae18c5cbbbddf2ee84d#diff-2fa6230b9e8601cd9e1fae2257f1c95f95e307ede63335c924646f90582eee34R91-R100 is missing.

Do not remove Vimscript code in new-style Vim tests that should be skipped in Neovim.